stock-tracker/package.json

```diff
@@ -41,7 +41,7 @@
   },
   "devDependencies": {
     "husky": "^3.0.8",
-    "nodemon": "^1.19.2",
+    "nodemon": "^1.19.4",
```

Comment:
This change has no relevant to the rest of the PR, and shouldn't be here.
I'll accept leaving it in for now, but try to remember not to bleed into other features / changes.